apps/www/pages/pricing/index.tsx

```diff
@@ -341,9 +341,11 @@ export default function IndexPage() {
           <div className="grid lg:grid-cols-3 gap-4 mb-16">
             {addons.map((addon) => (
               <div className="bg-white dark:bg-scale-300 rounded-[4px]" key={addon.name}>
-                <div className="overflow-hidden rounded-lg">
-                  <img
+                <div className="overflow-hidden rounded-lg relative h-56">
+                  <Image
```

Comment:
That seems to work, but you probably also need to update <Image .../> on lines 355 and 422. @saltcod 

<img width="1425" alt="Screenshot 2023-10-02 at 14 31 16" src="https://github.com/supabase/supabase/assets/25671831/bd74686d-a4cb-4dc9-ac77-6442e676d256">